lib/DebuggerAgent.js

```diff
@@ -105,7 +105,7 @@ DebuggerAgent.prototype = {
     this._debuggerClient.request(
       'scripts',
       {
-        includeSource: true,
+        includeSource: false,
```

Comment:
I don't know why we need to receive sources here.
I can revert this line or move it to isolated commit.